pkg/crawler/log.go

```diff
@@ -37,11 +52,12 @@ func (c *Crawler) log(statusCode int, url string, duration time.Duration) {
 // logError logs a json log with an error, url, timestamp and duration of the request
 func (c *Crawler) logError(err error, url string, duration time.Duration) {
 	message := fmt.Sprintf(
-		`{"err": %s, "url": "%s", "time": %d, "duration": %d}`,
-		escapeString(err.Error()),
+		`{"status": %d, "url": "%s", "time": %d, "duration": %d, "error": %s}`,
+		218,
```

Comment:
I actually thought it would be useful to return this common to all error messages. Especially a json output can be filtered very easily with this.

Resolved.